sql/log_event.cc

```diff
@@ -7187,6 +7187,11 @@ void Intvar_log_event::print(FILE* file, PRINT_EVENT_INFO* print_event_info)
   case INSERT_ID_EVENT:
     msg="INSERT_ID";
     break;
+#ifdef WITH_WSREP
+  case BINLOG_CONTROL_EVENT:
+    msg="BINLOG_CONTROL";
+    break;
```

Comment:
Can we add an assert here since BINLOG_CONTROL Intvar_event is not written to the binlog and this code will never be hit when running mysqlbinlog?

Reply:
done